lldb/source/Plugins/Process/minidump/MinidumpParser.h

```diff
@@ -45,9 +49,16 @@ struct Range {
   friend bool operator==(const Range &lhs, const Range &rhs) {
     return lhs.start == rhs.start && lhs.range_ref == rhs.range_ref;
   }
+
+  friend bool operator<(const Range &lhs, const Range &rhs) {
+    return lhs.start < rhs.start;
+  }
```

Comment:
You don't need this class anymore. Just store a `llvm::ArrayRef<uint8_t>` in the `MemoryRangeVector` definition instead of a `minidump::Range`